Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -2326,7 +2326,7 @@ static String convert_number_to_date_string(double input)
     // date string that represents the date that, in UTC, is current input milliseconds after midnight UTC
     // on the morning of 1970-01-01 (the time represented by the value "1970-01-01T00:00:00.0Z").
     auto date = AK::UnixDateTime::from_seconds_since_epoch(input / 1000.);
-    return MUST(date.to_string("%Y-%m-%d"sv));
+    return MUST(date.to_string("%Y-%m-%d"sv, AK::UnixDateTime::LocalTime::No));
```

Comment:
Can we have some kind of regression test for this? :)

Reply:
I added an `internals` method to switch the test's time zone similar to what we do in `test-js`